config/routes.rb

```diff
@@ -47,4 +47,5 @@
     GovukHealthcheck::Mongoid,
     GovukHealthcheck::SidekiqRedis,
   )
+  get "/whats-new" => "whats_new#index", as: :whats_new
```

Comment:
As we're installing govuk_publishing_components for the first time we should also add the component guide:

```
mount GovukPublishingComponents::Engine, at: "/component-guide" if Rails.env.development?
```

We may need an initializer too: https://github.com/alphagov/govuk_publishing_components/blob/main/docs/install-and-use.md#3-configure-the-gem

It'd be valuable to have a read of the documentation on GovukPublishingComponents, it explains various conventions and approaches that should be valuable for the work to convert to design system: https://github.com/alphagov/govuk_publishing_components#technical-documentation

Reply:
yep makes sense.  seems like following [this documentation on migration in govuk-frontend](https://frontend.design-system.service.gov.uk/migrating-from-legacy-products/#choose-a-template-engine) has caused a lot of confusion up to now.